code/styles/layout.css

```diff
@@ -2,6 +2,20 @@
   max-width: 100%;
 }
 
+header {
+  background-image: url("../image/Stars.jpg");
+}
+
```

Comment:
i looked through all of your css and i couldnt figure out why the background image doesnt do what it's supposed to